crates/propolis-api-types/src/lib.rs

```diff
@@ -396,6 +396,16 @@ pub struct VCRRequestPathParams {
     pub id: Uuid,
 }
 
+#[derive(Deserialize, JsonSchema)]
+pub struct VolumeStatusPathParams {
+    pub id: Uuid,
+}
+
+#[derive(Debug, Serialize, Deserialize, JsonSchema)]
+pub struct VolumeStatus {
+    pub active: bool,
```

Comment:
Do we foresee any other pieces of status information being useful in the future?

Reply:
None immediately come to mind. I could see cases where Nexus may want to query what Volume Construction Request is currently in the instance spec (to compare with the latest that is stored in CRDB), but it could just use `GET /instance/spec` in that case.